gravitee-apim-portal-webui-next/src/app/api-details/api-details.component.spec.ts

```diff
@@ -55,12 +55,18 @@ describe('ApiDetailsComponent', () => {
     expectPageList(fakePagesResponse({ data: [] }));
     const documentationTab = await harnessLoader.getHarness(MatTabHarness.with({ label: 'Documentation' }));
     expect(await documentationTab.isDisabled()).toEqual(true);
+
```

Comment:
If you'd like, you can add this expect to an `afterEach` before the `httpTestController.verify()`

Reply:
Done